roles/openstack_infra/tasks/cleanup.yml

```diff
@@ -0,0 +1,50 @@
+- name: delete master node
+  os_server:
+    name: master
+    state: absent
+    wait: yes
+- name: delete worker0
+  os_server:
+    name: worker0
+    state: absent
+    wait: yes
+- name: delete worker1
+  os_server:
+    name: worker1
+    state: absent
+    wait: yes
+- name: delete worker2
+  os_server:
+    name: worker2
+    state: absent
+    wait: yes
+- name: delete custom security group
+  os_security_group:
+    state: absent
+    name: manila-kube-security-group
+- name: delete keypair
+  os_keypair:
+    state: absent
+    name: "{{ key_name }}"
+    public_key_file: "{{ public_key_file }}"
```

Comment:
I don't think we need the public_key_file line for the removal.

Reply:
Done